dev/fattest.simplicity/bnd.bnd

```diff
@@ -75,19 +71,20 @@ fat.test.container.images: testcontainers/ryuk:0.6.0, testcontainers/sshd:1.1.0,
 -buildpath: \
     com.ibm.websphere.javaee.jaxb.2.2;version=latest,\
     commons-httpclient:commons-httpclient;version=3.1,\
-    com.ibm.websphere.javaee.jsonp.1.1;version=latest,\
+    com.ibm.websphere.javaee.jsonp.1.0;version=latest,\
     com.ibm.websphere.javaee.servlet.3.0;version=latest,\
     io.openliberty.jakarta.servlet.5.0;version=latest,\
     org.eclipse.transformer:org.eclipse.transformer.cli;version=0.5.0,\
     org.eclipse.transformer:org.eclipse.transformer.bnd.analyzer;version=0.5.0,\
     org.eclipse.transformer:org.eclipse.transformer.jakarta;version=0.5.0,\
     org.eclipse.transformer:org.eclipse.transformer;version=0.5.0,\
-    biz.aQute.bnd:biz.aQute.bnd.transform;version=7.0.0,\
-    commons-cli:commons-cli;version=latest,\
+    biz.aQute.bnd:biz.aQute.bnd.transform;strategy=exact;version=6.4.1,\
+    biz.aQute.bnd:biz.aQute.bnd.transform;strategy=exact;version=7.0.0,\
```

Comment:
I'm curious why it has both of these?

Reply:
7.0.0 requires Java 17.  So for Java 17 and Java 21, we use 7.0.0.  For Java 8 and 11 testing, we use 6.4.1.  6.4.1 can't be used with Java 21 due to a bug that they will not fix in bnd 6.4.1 level and require that we use 7.0.0 for the Java 21 bug.